pkg/auth/handler/webapp/authflowv2/change_password.go

```diff
@@ -57,6 +58,15 @@ func (h *AuthflowV2ChangePasswordHandler) GetData(w http.ResponseWriter, r *http
 			IsNew: false,
 		},
 	)
+
+	// put policy PasswordBelowGuessableLevel to front
+	for p, x := range passwordPolicyViewModel.PasswordPolicies {
+		if x.Name == password.PasswordBelowGuessableLevel {
+			passwordPolicyViewModel.PasswordPolicies = append([]password.Policy{x}, append((passwordPolicyViewModel.PasswordPolicies)[:p], (passwordPolicyViewModel.PasswordPolicies)[p+1:]...)...)
+			break
+		}
+	}
```

Comment:
It looks like a ui logic.
What if simply use `order-1` in the corresponding element?

Reply:
You mean access the array directly by index? e.g. `policy = policies[order-1]`

Seems it's will be easily broken if we assume the order of policies :derpthink:
And I think the policies length is not consistent?

Reply:
@Ngkaokis Sorry I just rechecked the doc and it should be `order-first`. It is a tailwind util.
https://tailwindcss.com/docs/order

I mean in the corresponding frontend element, add this class to make it always be the first item.